vnext/Shared/OInstance.cpp

```diff
@@ -112,8 +112,7 @@ class OJSIExecutorFactory : public JSExecutorFactory {
     TurboModuleBinding::install(
         *runtimeHolder_->getRuntime(),
         std::function(binding),
-        TurboModuleBindingMode::HostObject,
-        longLivedObjectCollection_);
```

Comment:
>longLivedObjectCollection_ [](http://example.com/codeflow?start=8&length=26)

We needed it to maintain lifetime of JSI objects. I wonder how does it work without it.